### ISSUE TITLE
[aria-*] attributes do not match their roles

### ISSUE THREAD

**Issue by catalinpit** [catalinpit/attila-theme#9]
The blog has the following accessibility issue.

<img width="960" alt="CleanShot 2023-12-29 at 17 25 21@2x" src="https://github.com/catalinpit/attila-theme/assets/25515812/daa93eb6-4476-470c-a012-2d2411a821b0">

**Comment by dhinrichs-scottlogic** [comment]
Hi, would love to help out with this if that's welcome? Let me know and I'll make a PR for it.

- My guess is that this warning shows up because the icon is understood as an image, and for images the the alt attribute would be better than the aria-label.
- However, in this case I'd recommend to actually hide the icon (since it's purely decorative), and give the link itself the aria-label to meet [2.4.4 link purpose](https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8.html). 
- Alternatively, you could add a visually hidden span (you'd need to add the [necessary css](https://blog.logrocket.com/design-accessibility-css-visually-hidden-class/) to hide it) to your links, like [in this example](https://a11y-101.com/development/icons-and-links). But just adding aria-label to the link will do the job as well. 
- I also noticed that the arrow to the left is currently labelled as "next post" and the arrow to the right as "previous post",  but I wonder if the other way round **(arrow left = previous, and arrow right = next) would be better?**

**Comment by catalinpit** [comment]
Hey @dhinrichs-scottlogic, your help would be very much appreciated! I'd leave it up to you to decide what's the most appropriate and efficient way for handling that.

> I also noticed that the arrow to the left is currently labelled as "next post" and the arrow to the right as "previous post", but I wonder if the other way round (arrow left = previous, and arrow right = next) would be better?

The behaviour of this is a bit weird. At the moment, the left arrow takes you to the next post and the right arrow takes you to the previous post. I think they should be the other way around - e.g. left arrow should take you to the previous post and the right arrow to the next post.

Should I assign this to you?

**Comment by dhinrichs-scottlogic** [comment]
Yes, I'll have a go at it
